doc/deployment.md

```diff
@@ -0,0 +1,51 @@
+# Production Deployment
+
+1. Ensure changelog is up to date
+- Do a commit log diff against the previous version, ensure all the important changes are in the upcoming release
+- Keep a blank upcoming release
+- Verify changelog with other teams
+ 
+2. Tag release
+- Create a tag off master with the same name as in the changelog. Typically with the format: yyyy-mm-dd-n.
+- Add a short description in the tag for the release
+- `git tag -a <yyyy-mm-dd-n> -m "<description>"`
+- `git push --tags`
+
+3. Understand migrations to be run as a part of this release
+- `git diff <previous-tag>..<current-tag> db/migrate`
+- lookout for data migrations, and irreversible migrations   
+
+Perform the following steps first on staging, QA it, and then perform them on production.
```

Comment:
Rather than have an iterative instruction, I'd explicitly call out that the next steps should be done on **staging**. Then, below all of them, have a line that says repeat the above for **production** if everything went OK.

Reply:
Cool, that sounds nicer. Will change.